objects/avaligeneric/lab/avalisample4.object

```diff
@@ -5,12 +5,12 @@
   "category" : "furniture",
   "price" : 35,
   "printable" : false,
-  "description" : "The crystal inside is quite interesting... Could it be a sync crystal?",
+  "description" : "The crystal inside is quite interesting... could it be a sync crystal?",
```

Comment:
Capitalize. Comma wouldn't make sense here (Semicolon may, but period makes the most sense)